TECHNIQUES_TO_PULL_DATA_FROM_EVALAI.md

```diff
@@ -0,0 +1,31 @@
+# Techniques to pull data from EvalAI within EvalAI-ngx without using 'git clone' in dockerfile
```

Comment:
Why is this required? Please let me know if I am missing something here.

Reply:
It's not. It was a mistake and it has been removed.byou can see if you check the branch files @vkartik97

Reply:
Oh! Sorry! Didn't notice it was still there. My sincere apologies @vkartik97